### ISSUE TITLE
Object.keys(parentModulePackage.scripts)

### ISSUE THREAD

**Issue by reggi** [PixnBits/githook-scripts#9]
Whats going on here?

```
/Users/treggi/Projects/my-project/node_modules/githook-scripts/postinstall.js:47
Object.keys(parentModulePackage.scripts)
       ^

TypeError: Cannot convert undefined or null to object
    at Object.<anonymous> (/Users/treggi/Projects/my-project/node_modules/githook-scripts/postinstall.js:47:8)
    at Module._compile (module.js:570:32)
    at Object.Module._extensions..js (module.js:579:10)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)
    at Module.runMain (module.js:604:10)
    at run (bootstrap_node.js:394:7)
    at startup (bootstrap_node.js:149:9)
    at bootstrap_node.js:509:3
npm WARN axp-app No repository field.
npm WARN axp-app No license field.
npm ERR! Darwin 16.7.0
npm ERR! argv "/Users/treggi/.nvm/versions/node/v6.8.0/bin/node" "/Users/treggi/.nvm/versions/node/v6.8.0/bin/npm" "install"
npm ERR! node v6.8.0
npm ERR! npm  v3.10.8
npm ERR! code ELIFECYCLE

npm ERR! githook-scripts@1.0.0 postinstall: `node postinstall.js`
```

**Comment by PixnBits** [comment]
Looks like I'm not checking for the case where `scripts` isn't defined in the `package.json`. I can add that as a bug fix, but FWIW `npm init` will add a test script by default.

**Comment by reggi** [comment]
It happens to be that this error is a private dependency I am using. Any idea why when I install this would be happen?

**Comment by reggi** [comment]
@PixnBits here's the folder structure that I am working with. May have something to do with the CWD.

```
➜  test-github-scripts git:(master) tree
.
├── lerna-debug.log
├── lerna.json
├── package.json
└── packages
    └── example
        ├── npm-debug.log
        └── package.json
```

**Comment by PixnBits** [comment]
Ah, monorepo! Is `githook-scripts` listed as a devDep of the root `package.json`? Does that root package have a scripts object?

**Comment by PixnBits** [comment]
fix committed in #11 
published as v1.0.1
